gateway/sds_gateway/api_methods/management/commands/reset_index.py

```diff
@@ -0,0 +1,494 @@
+"""Django management command to initialize OpenSearch indices."""
+
+from datetime import UTC
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+from opensearchpy import AuthenticationException
+from opensearchpy import ConnectionError as OpensearchConnectionError
+from opensearchpy import NotFoundError
+from opensearchpy import OpenSearch
+from opensearchpy import RequestError
+
+from sds_gateway.api_methods.models import Capture
+from sds_gateway.api_methods.models import CaptureType
+from sds_gateway.api_methods.utils.metadata_schemas import get_mapping_by_capture_type
+from sds_gateway.api_methods.utils.opensearch_client import get_opensearch_client
+from sds_gateway.api_methods.views.capture_endpoints import CaptureViewSet
+
+
+class Command(BaseCommand):
+    """Initialize OpenSearch indices for different capture types."""
+
```

Comment:
I think we should add some documentation / guidance to use this command:

1. When should we run `reset_index`;
2. Risks and what to do before running it (e.g. backup OpenSearch and RDBMS);
3. What exactly it does e.g.: deletes invalid captures; re-creates the OpenSearch indices; re-index the remaining ones; "on failure of X, it will roll back Y".
4. What to do on failure when crying doesn't help;
5. Which of these operations can be manually rolled back, and how.